client/homebrew/editor/tagInput/tagInput.jsx

```diff
@@ -0,0 +1,105 @@
+require('./tagInput.less');
+const React = require('react');
+const { useState, useEffect } = React;
+const _     = require('lodash');
+
+const TagInput = ({ unique = true, values = [], ...props }) => {
+	const [temporaryValue, setTemporaryValue] = useState('');
```

Comment:
I might name this `inputText` or something to convey what content this value represents.